utils/src/fhir-migration.ts

```diff
@@ -0,0 +1,200 @@
+import * as dotenv from "dotenv";
+dotenv.config();
+
+import { Sequelize } from "sequelize";
+import { MedplumClient } from "@medplum/core";
+import { Organization as FHIROrganization, Patient as FHIRPatient } from "@medplum/fhirtypes";
+import { PersonalIdentifier } from "@metriport/api";
+
+/**
+ * Migrate existing orgs and patients to FHIR
+ *
+ * Requires env vars as indicated at start of this function.
+ * Its suggested to set those on a .env file so they are not stored on
+ * your shell history.
+ *
+ * Created as part of #521
+ */
+async function main() {
+  const sqlDBCreds = getEnvVarOrFail("DB_CREDS");
+  const fhirUrl = getEnvVarOrFail("FHIR_URL");
+
+  const dbCreds = JSON.parse(sqlDBCreds);
+
+  const fhirApi = new MedplumClient({
+    baseUrl: fhirUrl,
+    fhirUrlPath: "fhir",
+  });
+
+  const sequelize = new Sequelize(dbCreds.dbname, dbCreds.username, dbCreds.password, {
+    host: dbCreds.host,
+    port: dbCreds.port,
+    dialect: dbCreds.engine,
+  });
+
+  const orgResults = await sequelize.query("SELECT * FROM organization");
+  const organizations = orgResults[0];
+
+  organizations.forEach(async org => {
+    const fhirOrg = toFHIROrg(org);
+    await fhirApi.updateResource(fhirOrg);
+  });
+
+  const patientResults = await sequelize.query("SELECT * FROM patient");
+  const patients = patientResults[0];
+
+  patients.forEach(async patient => {
+    const fhirPatient = toFHIRPatient(patient);
+    await fhirApi.updateResource(fhirPatient);
+  });
+
+  console.log(`Done`);
+}
+
+function getEnvVar(varName: string): string | undefined {
+  return process.env[varName];
+}
+function getEnvVarOrFail(varName: string): string {
+  const value = getEnvVar(varName);
+  if (!value || value.trim().length < 1) {
+    throw new Error(`Missing ${varName} env var`);
+  }
+  return value;
+}
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const toFHIROrg = (org: any): FHIROrganization => {
```

Comment:
This disable doesnt work in the github build, the results from sequel arent typed so what would be the best approach to fixing this type issue besides make the file a js file

Reply:
I can't tell what the issue is by your comment, can you point to the failed job?

It does work to build local, right?

Reply:
Yea it works locally this is the error https://github.com/metriport/metriport/actions/runs/4701736182/jobs/8338150898?pr=223

Reply:
Additionally, about TS referring to stuff from parent folder (mimicking Node's): https://github.com/microsoft/TypeScript/issues/30124

Reply:
Yea theres no more issue with this